src/login/logind-button.h

```diff
@@ -5,6 +5,22 @@ typedef struct Button Button;
 
 #include "logind.h"
 
+typedef enum ButtonModifierMask {
+        BUTTON_MODIFIER_NONE        = 0,
+        BUTTON_MODIFIER_LEFT_SHIFT  = 1 << 0,
+        BUTTON_MODIFIER_RIGHT_SHIFT = 1 << 1,
+        BUTTON_MODIFIER_LEFT_CTRL   = 1 << 2,
+        BUTTON_MODIFIER_RIGHT_CTRL  = 1 << 3,
+        BUTTON_MODIFIER_LEFT_ALT    = 1 << 4,
+        BUTTON_MODIFIER_RIGHT_ALT   = 1 << 5,
+        _BUTTON_MODIFIER_MAX        = 1 << 6,
```

Comment:
this one doesn't make sense. for flags enums it might make sense to define _BUTTON_MODIFIER_MAX as all bits on (i.e. here: `(1 << 6)-1`) but frankly, i'd not bother here, you are not using that.